include/input/HebiKeyboard.m

```diff
@@ -85,9 +87,27 @@ function loadLibs()
             
         end
         
-        function out = read(this)
+        function [keys, diffKeys] = read(this)
```

Comment:
This MATLAB feature is super nice for backwards compatibility...

Reply:
I agree 👍